src/storage-types/src/sources.rs

```diff
@@ -1317,7 +1324,23 @@ mod tests {
     fn all_scalar_types_columnar_roundtrip() {
```

Comment:
nit: Should this and the other test name be s/columnar/parquet/ now that we're using `parquet::validate_roundtrip` instead of `columnar::validate_roundtrip`?

Reply:
Good idea, updated!